client/Lykke.Service.PayInternal.Client/PayInternalClient.cs

```diff
@@ -20,6 +21,8 @@ public class PayInternalClient : IPayInternalClient, IDisposable
         private readonly IMerchantsApi _merchantsApi;
         private readonly IOrdersApi _ordersApi;
         private readonly IPaymentRequestsApi _paymentRequestsApi;
+        private readonly ITransferReportApi _transferReportApi;
```

Comment:
Where is new API's implementation?

Reply:
Actually fixed by adding the appropriate code to client classes.